src/components/cc-order-summary/cc-order-summary.js

```diff
@@ -0,0 +1,233 @@
+ * - a list of details for additional information.
+ *
+ * @cssdisplay block
+ *
+ * @fires {CustomEvent} cc-order-summary:create - Fires when the create button is clicked.
+ *
+ * @cssprop {FontSize} --cc-order-summary-detail-font-size - The font-size for the list of details (defaults: `0.825em`).
+ * @cssprop {FontWeight} --cc-order-summary-font-weight - Sets the value of the font weight CSS property (defaults: `600`).
+ *
+ * @slot detail - a single piece of information displayed under the main part of the UI. You can insert multiple detail items.
+ */
+
+export class CcOrderSummary extends LitElement {
+  static get properties() {
+    return {
+      orderSummaryState: { type: Object, attribute: 'order-summary-state' },
+    };
+  }
+
+  constructor() {
+    super();
+
+    /** @type {OrderSummaryState} component main state */
+    this.orderSummaryState = null;
+  }
+
+  _onCreateClick() {
+    dispatchCustomEvent(this, 'create');
+  }
+
+  render() {
+    if (this.orderSummaryState == null) {
+      return '';
+    }
+
+    return html`
+      <div class="title">${i18n('cc-order-summary.title')}</div>
+      <div class="summary">${this._renderSummary()}</div>
+      <div class="details-container">
+        <slot name="detail"></slot>
+      </div>
+    `;
+  }
+
+  _renderSummary() {
+    return html`
+      <div class="header">${this._renderSummaryHeader()}</div>
+      <div class="body">${this._renderSummaryBody()}</div>
+      <div class="footer">
+        <cc-button class="btn-submit" type="submit" primary @cc-button:click=${this._onCreateClick}>
+          ${i18n('cc-order-summary.create')}
+        </cc-button>
+      </div>
+    `;
+  }
+
+  _renderSummaryHeader() {
+    return html`${this.orderSummaryState.name != null && this.orderSummaryState.name !== ''
+      ? html`<div class="header--name">${this.orderSummaryState.name}</div>`
+      : html`<div class="header--name header--name-empty">&hellip;</div>`}
+    ${this.orderSummaryState.tags?.length > 0
+      ? html`
+          <div class="header--tags">
+            ${this.orderSummaryState.tags
+              .filter((tag) => tag != null && tag.trim() !== '')
+              .map((tag) => html`<cc-badge intent="info" weight="dimmed">${tag.trim()}</cc-badge>`)}
+          </div>
+        `
+      : ``}
+    ${this.orderSummaryState.logoUrl != null
+      ? html`<div class="header--logo">
+          <cc-img class="logo" src="${this.orderSummaryState.logoUrl}" title="${this.orderSummaryState.name}"></cc-img>
+        </div>`
+      : ``}`;
+  }
+
+  _renderSummaryBody() {
+    return html`
+      ${this.orderSummaryState.configuration?.length > 0
+        ? this.orderSummaryState.configuration.map((configItem) => {
+            const { label, value } = configItem;
+            return html`<div class="body--item">
+              <div class="body--label">${label}</div>
+              <div class="body--value">${value}</div>
+            </div>`;
+          })
+        : ``}
+    `;
```

Comment:
nitpick: we could simplify this like that

```suggestion
    return this.orderSummaryState.configuration?.map((configItem) => {
        const { label, value } = configItem;
        return html`<div class="body--item">
          <div class="body--label">${label}</div>
          <div class="body--value">${value}</div>
        </div>`;
    });
```